slidechain/slidechain_test.go

```diff
@@ -294,178 +302,183 @@ func TestEndToEnd(t *testing.T) {
 		}
 		c.launch(ctx)
 
-		// Prepare Stellar account to peg-in funds and txvm account to receive funds.
-		amount := 5 * xlm.Lumen
-		exporterPub, exporterPrv, err := ed25519.GenerateKey(nil)
-		if err != nil {
-			t.Fatalf("error generating txvm recipient keypair: %s", err)
-		}
-		var exporterSeed [32]byte
-		copy(exporterSeed[:], exporterPrv)
-		exporter, err := keypair.FromRawSeed(exporterSeed)
-		err = stellar.FundAccount(exporter.Address())
-		if err != nil {
-			t.Fatalf("error funding account %s: %s", exporter.Address(), err)
-		}
+		for _, tt := range tests {
+			// Prepare Stellar account to peg-in funds and txvm account to receive funds.
+			inputAmount := tt.inputAmount
+			exportAmount := tt.exportAmount
+			exporterPub, exporterPrv, err := ed25519.GenerateKey(nil)
+			if err != nil {
+				t.Fatalf("error generating txvm recipient keypair: %s", err)
+			}
+			var exporterSeed [32]byte
+			copy(exporterSeed[:], exporterPrv)
+			exporter, err := keypair.FromRawSeed(exporterSeed)
+			err = stellar.FundAccount(exporter.Address())
```

Comment:
Could we also move this exporter account creation + funding outside of the test loop?